index.js

```diff
@@ -148,8 +148,12 @@ SlackBot.prototype.buildRouter = function () {
   return function (event, context, callback) {
     var foundCommand;
     var builtEvent = event;
-    builtEvent.body = qs.parse(builtEvent.body);
 
+    if (builtEvent.body === 'ping') {
+      return context.succeed('Ok');
+    }
```

Comment:
I'm not sure how we could make it faster, this is pretty much at the beginning of the handler. I think $17/year is worth not having to wait.